pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio reproduced by the [web page][] changes.
+
+If the [web page][] is reproducing time-based media, rendered pixels and reproduced audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered and reproduced if the event is not fired, with the ones that are rendered if the event is fired.
+
+For the purpose of this definition, it is assumed that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event.
```

Comment:
Wouldn't the second assumption de facto exclude time-based media? (thus making the first one useless)
"the page does not change if there is no such event" => no time based media is played.

Reply:
I don't think so. Take this scenario
1 - The user activates play and the web page starts playing audio in the background (there was a change in content but for the purpose of this rule the event that caused it did not originate from device motion)
2 - The user flicks the device to the right to move to the next track (there was another change in content - this time originated by a device motion event - even though the audio was already playing)

The first assumption kicks in in the second part of this scenario and it is what allows us to detect the change. 

To be honest, I'm still not totally convinced that we need the second one. I can't really come up with an example of something that will cause changes to the content without being event-based.